lib/levels/enemies/enemies-one.js

```diff
@@ -2,9 +2,9 @@ const SimpleEnemy = require('../../simple-enemy');
 const QuickEnemy = require('../../quick-enemy');
 
 var enemiesOne = [
```

Comment:
This somehow is the problem. I've done more debugging, and the issue is definitely that in `get-enemies.js`, the first time the game is loaded, it calls `enemies[1]` (since the level is one) and loads the file `enemies-one.js` which we are linked to here. You can see that in this file, we create 3 brand new enemies. 

However, the second time around after you click to go to the next  difficulty, `enemies[1]` now returns 3 enemies, but the x values are off by a non-constant amount (enemy1 goes to x: 210, enemy2 goes to x: 346, and enemy3 goes to x: 407) and all of their alive values are false. Since it is calling the same file `enemies-one.js` I have no idea how this is happening.